### ISSUE TITLE
Can't get it to work!

### ISSUE THREAD

**Issue by crystallove18** [davidjbradshaw/image-map-resizer#22]
I don't understand why it doesn't work for me... am I missing something?

**Comment by davidjbradshaw** [comment]
I expect so, but as you haven't provided a link to you code or a copy of the browser error console. I can't comment.

**Comment by crystallove18** [comment]
Hi,
I’m sorry about not providing information. This is the link: www.techspa.ca/garza
This is the code:

<!doctype html>
<html class="no-js" lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Contact Information | Daniel Garza, MD Psychiatry | New York City</title>
    <link rel="stylesheet" href="css/foundation.css" />
    <link rel="stylesheet" href="css/style.css" />
    <script src="js/vendor/modernizr.js"></script>
<link href='http://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=EB+Garamond' rel='stylesheet' type='text/css'>
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
<script src="js/jquery.rwdImageMaps.min.js"></script>
<script src="image-map-resizer-master/js/imageMapResizer.min.js"></script>
</head>
  <body>
    
           

      <div class="row"  style="padding-left:0">
        <div class="large-12 columns header"  style="padding-left:0">
        	<div class="large-3 columns show-for-large-only" style="padding-left:0">
        		<img src="img/city-street.jpg" alt="Daniel Garza, MD Psychiatry"/>
        	</div>
        	<div class="large-9 columns">
          <div class="nav-bar right show-for-large-only">
           <ul>
             <li><a title="Home" href="index.html"><img src="img/home-link.png"/></a></li>
             <li><a title="Info" class="current"  href="info.html"><img src="img/info-link-current.png"/></a></li>
             <li><a title="About" href="about.html"><img src="img/about-link.png"/></a></li>
            </ul>
          </div>
          
          <div class="nav-bar show-for-small-only">
           <ul>
             <li><a title="Home" href="index.html"><img   style="width:66px;" src="img/home-link.png"/></a></li>
              <li><a title="Info" class="current"  href="info.html"><img   style="width:66px;" src="img/info-link-current.png"/></a></li>
             <li><a title="About" href="about.html"><img   style="width:66px;" src="img/about-link.png"/></a></li>
            </ul>
          </div>
          <div class="nav-bar show-for-medium-only">
           <ul>
             <li><a title="Home" href="index.html"><img src="img/home-link.png"/></a></li>
              <li><a title="Info" class="current"  href="info.html"><img src="img/info-link-current.png"/></a></li>
             <li><a title="About" href="about.html"><img src="img/about-link.png"/></a></li>
            </ul>
          </div>
		<h1 class="title"><img src="img/garza-title.png"/></h1>
           <h1 class="title show-for-sr">Daniel Garza, MD Psychiatry</h1>
         
         </div>

        </div>
      </div>
      <div class="row content2">
     <aside class="large-3 columns">
     </aside>
         
        <div class="large-9 columns" role="content">
     
       <img class="show-for-large-only" src="img/info-body.png" usemap="#Mail"/>
       <map id="Mail" name="Mail">
         <area shape="rect" coords="245,39,505,72" href="mailto:info@danielgarzamd.com">
         <area shape="rect" coords="105,94,658,126" href="#">
       </map>
      <img class="show-for-medium-only" src="img/mobile/info-body.png" usemap="#Mail"/>
      <img class="show-for-small-only" src="img/mobile/info-body.png" usemap="#Mail"/>
      
       <img class="show-for-small-only" src="img/city-street.jpg" alt="Daniel Garza, MD Psychiatry"/>
       <img class="show-for-medium-only" src="img/city-street.jpg" alt="Daniel Garza, MD Psychiatry"/>
 <p class="body-text show-for-sr">Appointments can be made by calling Dr. Garza at <span class="emphasis">646-641-5564</span> 
or by sending an email to <span class="emphasis" style="text-transform:lowercase; text-decoration:none;"><a href="mailto:info@danielgarzamd.com?subject=Demand of information from website">info@danielgarzamd.com</a></span>.</p>
<p class="body-text show-for-sr"><span class="emphasis">Address:</span> 330 W. 58th Street, Suite 203; New York, NY  10019 <a target="_blank" href="https://www.google.ca/maps/place/330+W+58th+St+%23203,+New+York,+NY+10019,+USA/@40.7677324,-73.9836128,17z/data=!3m1!4b1!4m2!3m1!1s0x89c258588652b53b:0x4146c1671e0ce786?hl=en" style="color:#CD8458">VIEW MAP</a></p>
<p class="body-text show-for-sr"><span class="emphasis">DIRECTIONS:</span> The office is located on 58th Street between 8th and 9th Avenues.</p> 
<p class="body-text show-for-sr">It is adjacent to the 59th Street Columbus Circle subway station. 
Parking is available at various commercial lots located and on 57th and 58th Streets and more expensively across the street at the 
Time Warner Center.</p>
<p class="body-text show-for-sr"><span class="emphasis">INSURANCE:</span> Insurance is not accepted, though many individuals are reimbursed for the cost of visits by the out-of-network benefits component of their existing health insurance plans.</p>
        
        </div>    
      </div>
      
      
      <footer class="row">
        <div class="large-12 columns">
          <div class="row">
            <div class="large-6 columns">
              <p class="body-text">© Daniel Garza, MD Psychiatry 2015</p>
            </div>
            <div class="large-6 columns">
              <ul class="right footer-link">
                <li>Website by: <a target="_blank" href="http://www.techspa.ca">TECHSPA, INC</a></li>
              </ul>
            </div>
          </div>
        </div>
      </footer>
    
    
    <script src="js/vendor/jquery.js"></script>
    <script src="js/foundation.min.js"></script>
    <script>
      $(document).foundation();
 </script>
<script> $(document).ready(function() {
    $('map').imageMapResize();
});</script>
  </body>
</html>
